### ISSUE TITLE
Fix all eslint errors in the application

### ISSUE THREAD

**Issue by nickytonline** [drewclem/protege#188]
## Goal

Now that we have the configuration being picked up thanks to #185, we can start fixing eslint issues. This is an all encompassing issues for eslint errors.

## Reason

When you run eslint, there are over 200 lint errors.

### Tips

* First step is to run `node_modules/.bin/eslint ./src/**/*.js --fix` or use `npx eslint ./src/**/*.js --fix` to automatically fix any errors that eslint is capable of fixing.
* It probably makes sense to have a PR that fixes the eslint errors for each page instead of one massive PR for fixing all the errors. It will be easier to grok and review.

## Definition of Done

There are no more eslint errors and the application functions just as it did before the lint errors were fixed.

**Comment by nickytonline** [comment]
Now that the eslint configuration is being picked up properly, there are errors in regards to missing prop types. You will need to run `npm install prop-types` to have access to prop types as they do not ship out of the box with Create React App (CRA).

One of the props is a job entity. I have a common proptype that can be reused wherever a job entity prop type exists.

```javascript
import PropTypes from ‘prop-types’

const jobPropType = PropTypes.shape({
  id: PropTypes.number.isRequired,
  companyName: PropTypes.string.isRequired,
  companyLogo: PropTypes.string.isRequired,
  jobTitle: PropTypes.string.isRequired,
  roleFocus: PropTypes.oneOf([‘Front-end’, ‘Back-end’, ‘Full-stack’])
    .isRequired,
  postedAt: PropTypes.shape({
    nanoseconds: PropTypes.number.isRequired,
    seconds: PropTypes.number.isRequired,
    toDate: PropTypes.func.isRequired,
  }).isRequired,
})

export default jobPropType
```

The `postedAt` property is a FireStore TimeStamp object. It could also use the [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes) `instanceOf]()`, 
 e.g. `PropTypes.instanceOf([TimeStamp](https://firebase.google.com/docs/reference/js/firebase.firestore.Timestamp))`.

**Comment by drewclem** [comment]
Solid issue @nickytonline. I'll definitely chip away at this one as I can.

Our husky pre-commit hooks will fail any commit while these errors exist. Is there a way to disable that part of the process while these are getting resolved? From the command line? VScode git integration?

**Comment by nickytonline** [comment]
@drewclem, while we're fixing eslint issues, you can bypass husky.

When committing code, run e.g. `git commit "my awesome message" --no-verify`

**Comment by drewclem** [comment]
Closed with #194 #195 #196
